evolve_keyboard_layout_wasm/www/app.js

```diff
@@ -48,13 +48,13 @@ Vue.component('evaluator-app', {
       </b-button>
 
       <b-button-group class="float-right">
-        <b-button :disabled="optStep > 0 || loading > 0" @click="optimizeInput" variant="secondary">
+        <b-button :disabled="optStep > 0 || loading > 0" @click="startOptimization" variant="secondary">
           <div v-if="optStep > 0 || loading > 0">
             <b-spinner small></b-spinner>
             <span v-if="optStep > 0">Iteration {{optStep}}/{{optTotalSteps}}</span>
             <span v-else>Loading</span>
           </div>
-          <div v-else>Optimize</div>
+          <div v-else>Optimize using {{ optMode }}</div>
```

Comment:
I would prefer keeping the button "simple" (without specifying the optimization method). 

I suggest having a `select` in the settings under the `Optimization` tab. Similarly as for the ngrams. Depending on the selection, the `genevo` or `sa` config file can be edited below the `select`

Reply:
I returned it to the original state.